### DISCUSSION TITLE
How to close all open tabs in a script?

### DISCUSSION THREAD

**Discussion by dhruvdcoder** [windingwind/zotero-actions-tags#295] [Q&A] [answered]
I could not find the API documentation for Zotero_Tabs object. How should I close all open reader tabs using a script?

**Comment by windingwind** [comment, marked as answer]
`Zotero_Tabs.closeAll()`


https://github.com/zotero/zotero/blob/0419c33e3d57c0c2cb0e7e4cc90b256c67ce89f8/chrome/content/zotero/tabs.js#L340-L345
